packages/data/README.md

```diff
@@ -174,11 +174,16 @@ const reduxStore = createStore();
 
 const boundSelectors = mapValues(
 	existingSelectors,
-	( selector ) => ( ...args ) => selector( reduxStore.getState(), ...args )
+	( selector ) =>
```

Comment:
ESLint likes to format code in a different way now and it picked up README.md for some reason